sim/battle.ts

```diff
@@ -1909,6 +1909,10 @@ export class Battle {
 			this.checkEVBalance();
 		}
 
+		if (format.customRules) {
+			this.add(`raw|<div class="broadcast-blue"><b>Custom challenges applied for this battle: ${format.customRules.join(', ')}</div>`);
```

Comment:
This probably should say custom rules, not custom challenges; you also forgot to close your <b> tag.

Reply:
screaming